tests/EdFi.Ods.WebApi.CompositeSpecFlowTests/Steps.cs

```diff
@@ -114,7 +114,10 @@ public async Task GivenTheSubjectOfTheRequestIsAStudentSchoolAssociation()
         {
             const string query = @"
                     SELECT  TOP 1 sa.Id
-                    FROM    edfi.StudentSchoolAssociation sa ";
+                    FROM    edfi.StudentSchoolAssociation sa
+                    INNER JOIN edfi.School sc 
+                            ON sa.SchoolId = sc.SchoolId
+                    WHERE sc.LocalEducationAgencyId IS NOT NULL";
```

Comment:
@bptillman I was thinking we would tap into the EducationOrganizationIdToEducationOrganizationId authorization view to identify a StudentSchoolAssociation that is accessible to the test client here. With the `IS NOT NULL` check that you've added here, It is still conceivable that we could end up with the same problem if another school was created for testing purposes like the one we saw in the test ODS for SchoolId = 7 except with the LocalEducationAgencyId set.

What do you think about making this a more durable query -- something like the following:

```sql
SELECT  TOP 1 ssa.Id
FROM    edfi.StudentSchoolAssociation ssa
        INNER JOIN auth.EducationOrganizationIdToEducationOrganizationId edOrgs
            ON ssa.SchoolId = edOrgs.TargetEducationOrganizationId
WHERE edOrgs.SourceEducationOrganizationId = 255901
```